pkg/cmd/bootstrap.go

```diff
@@ -171,23 +181,30 @@ func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client) er
 	}
 	io.GitOpsWebhookSecret = ui.EnterGitWebhookSecret()
 	io.ServiceRepoURL = utility.AddGitSuffixIfNecessary(ui.EnterServiceRepoURL())
-	if ui.IsPrivateRepo() {
-		io.GitHostAccessToken = ui.EnterGitHostAccessToken(io.ServiceRepoURL)
-	}
 	io.ServiceWebhookSecret = ui.EnterServiceWebhookSecret()
-	commitStatusTrackerCheck := ui.SelectOptionCommitStatusTracker()
-	if commitStatusTrackerCheck == "yes" {
-		io.CommitStatusTracker = true
-		if io.GitHostAccessToken == "" {
-			io.GitHostAccessToken = ui.EnterGitHostAccessToken(io.ServiceRepoURL)
+	secret, err := webhook.GetAccessToken(io.ServiceRepoURL)
+	if err != nil {
+		return err
+	}
+	if err == nil && secret == "" {
+		io.GitHostAccessToken = ui.EnterGitHostAccessToken(io.ServiceRepoURL)
```

Comment:
We should fix the help message of to get access token.   It is used by more than commit status tracker.